python/api_keys.txt

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
These are *your* keys and shouldn't be part of the pull request - in fact since they're now public here you should immediately revoke them in the developer portal and make sure that this file isn't part of a future pull request.

Reply:
Ya sorry, I forgot to take them out before uploading. I will revoke them and get new ones.